### DISCUSSION TITLE
Usage Capability is not working

### DISCUSSION THREAD

**Discussion by maeyounes** [open-webui/open-webui#6269] [General]
# Bug Report

## Installation Method

Docker

## Environment

- **Open WebUI Version:** v0.3.32

- **Operating System:** Ubuntu 22.04

**Confirmation:**

- [x] I have read and followed all the instructions provided in the README.md.
- [x] I am on the latest version of both Open WebUI and Ollama.
- [x] I have included the browser console logs.
- [x] I have included the Docker container logs.
- [x] I have provided the exact steps to reproduce the bug in the "Steps to Reproduce" section below.

## Expected Behavior:

Enabling Usage in Capabilities list should give statistics for the output such as number of tokens generated ... There should be a button next to the output options that show them when hovering or clicking.
## Actual Behavior:

No statistics are generated. The information/statistics button disappeared after recent updates.

## Description

**Bug Summary:**
In previous updates, there was an option to show statistics about the model generation, like token count for the input and output. This allows backends other than Ollama to show this information. After the recent updates, this option no longer works although ticked in the model's menu.

## Reproduction Details

**Steps to Reproduce:**
1. Use a different backend that Ollama such as vLLM.
2. Go to model menu in workspace. Enable usage under capabilities.
3. Ask a model and wait for the output generation.
4. The generated answer does not have the button that shows statistics as before.

## Logs and Screenshots

**Browser Console Logs:**
```
{
    "id": "chat-f01c2a6b7be04f7d82e1a6993b998d98",
    "object": "chat.completion.chunk",
    "created": 1729264126,
    "model": "Qwen/Qwen2.5-72B-Instruct-AWQ",
    "choices": [
        {
            "index": 0,
            "delta": {
                "content": " most"
            },
            "logprobs": null,
            "finish_reason": null
        }
    ],
    "usage": {
        "prompt_tokens": 39,
        "total_tokens": 40,
        "completion_tokens": 1
    }
}

```
**Docker Container Logs:**
```
harbor.webui  | INFO:     128.93.176.68:0 - "POST /api/v1/chats/tags HTTP/1.1" 200 OK
harbor.webui  | INFO:     128.93.176.68:0 - "GET /api/v1/users/user/settings HTTP/1.1" 200 OK
harbor.webui  | INFO:     128.93.176.68:0 - "POST /api/v1/users/user/settings/update HTTP/1.1" 200 OK
harbor.webui  | INFO:     128.93.176.68:0 - "GET /static/favicon.png HTTP/1.1" 304 Not Modified
harbor.webui  | INFO:     128.93.176.68:0 - "GET /static/favicon.png HTTP/1.1" 304 Not Modified
harbor.webui  | INFO:     128.93.176.68:0 - "POST /api/v1/chats/new HTTP/1.1" 200 OK
harbor.webui  | INFO:     128.93.176.68:0 - "GET /api/v1/chats/?page=1 HTTP/1.1" 200 OK
harbor.webui  | INFO:     128.93.176.68:0 - "POST /api/chat/completions HTTP/1.1" 200 OK
harbor.webui  | INFO:     128.93.176.68:0 - "POST /api/v1/chats/24fb8975-c64e-4c0a-bdf1-52f6c7cecf5e HTTP/1.1" 200 OK
harbor.webui  | INFO:     128.93.176.68:0 - "GET /api/v1/chats/?page=1 HTTP/1.1" 200 OK
harbor.webui  | INFO:     128.93.176.68:0 - "POST /api/chat/completed HTTP/1.1" 200 OK
harbor.webui  | INFO:     128.93.176.68:0 - "POST /api/v1/chats/24fb8975-c64e-4c0a-bdf1-52f6c7cecf5e HTTP/1.1" 200 OK
harbor.webui  | INFO:     128.93.176.68:0 - "GET /api/v1/chats/?page=1 HTTP/1.1" 200 OK
harbor.webui  | generate_title
harbor.webui  | Qwen/Qwen2.5-72B-Instruct-AWQ
harbor.webui  | INFO:     128.93.176.68:0 - "POST /api/task/title/completions HTTP/1.1" 200 OK
harbor.webui  | INFO:     128.93.176.68:0 - "POST /api/v1/chats/24fb8975-c64e-4c0a-bdf1-52f6c7cecf5e HTTP/1.1" 200 OK
harbor.webui  | INFO:     128.93.176.68:0 - "GET /api/v1/chats/?page=1 HTTP/1.1" 200 OK
harbor.webui  | INFO:     128.93.176.68:0 - "GET /api/v1/chats/?page=1 HTTP/1.1" 200 OK

```

**Screenshots/Screen Recordings (if applicable):**
![image](https://github.com/user-attachments/assets/09bd910f-10c9-4096-b210-18252d495753)
![image](https://github.com/user-attachments/assets/4d6afb03-52bd-4b87-8526-a7967393df44)

**Comment by AbdullahMPrograms** [comment]
I'm still facing this issue with a llama.cpp backend, I would love to see this working again!
